graphql-dgs-spring-boot-micrometer/src/main/kotlin/com/netflix/graphql/dgs/metrics/micrometer/DgsGraphQLMicrometerAutoConfiguration.kt

```diff
@@ -40,7 +40,6 @@ open class DgsGraphQLMicrometerAutoConfiguration {
     }
 
     @Bean
-    @Order(PriorityOrdered.LOWEST_PRECEDENCE)
```

Comment:
What does this fix? The `@Order` is just bean creation order, so I want to make sure what the intent is.

Reply:
I added this is a previous commit, which I now removed because it doesn't do anything. The micrometer test started failing because the order in which the instrumentations run seems to have changed in my branch. If the DgsGraphQlJavaErrorInstrumentation runs before the MetricsInstrumentation, the tag value is set properly (which is in this branch). However, if it's the other way around, the metrics add the tag for `gql.error` before the DgsGraphQlJavaErrorInstrumentation runs and that's why it passes in master branch.

Basically need a way to ensure the MetricsInstrumentation runs last, but not sure what the best way to do that is, so looking into it.

Reply:
Fixed by specifying order on both instrumentation classes. That ensures the correct relative ordering.